sjb/config/test_cases/test_pull_request_openshift_ansible_logging_37.yml

```diff
@@ -5,3 +5,4 @@ overrides:
     - "openshift,origin=release-3.7"
     - "openshift,aos-cd-jobs=master"
     - "openshift,origin-aggregated-logging=release-3.7"
+  evars: "-e openshift_disable_check=* -e openshift_install_examples=false"
```

Comment:
if `test_cases/test_branch_openshift_ansible_logging.yml` exposes these, I don't think you will need them in each child?